### ISSUE TITLE
Video accessibility

### ISSUE THREAD

**Issue by amandadupell** [grommet/grommet#6021]
### Expected Behavior

There are multiple aspects of the video component that appear to be broken or not accessible:

- [x] 1. User should be able to scrub along the timeline of the video to change the time position
- [x] 2. User plays or pauses the video via the extended "controls" prop. They are able to pause or play with any of the two controls. For example, the video will be playing and the user can go to the extended controls and hit the play icon. This will pause the video.

- [x] 3. The user cannot toggle captions on/off even if there is a designated `<track />` element with a captions file and "captions" is added to controls.

### Actual Behavior

There are multiple aspects of the video component that appear to be broken or not accessible:
1. User is unable to use their keyboard to scrub along the timeline of the video
2. There is a disconnect when controls include pause/play in the pop-up menu
3. Captions are not being supported even when including a `<track />` element with a captions/subtitles file (.vtt, .srt, etc.)

### URL, screen shot, or Codepen exhibiting the issue

[Codebox](https://codesandbox.io/s/laughing-franklin-9ge06h?file=/index.js)

### Your Environment

- Grommet version:
- Browser Name and version:
- Operating System and version (desktop or mobile):
